environment/src/main/java/com/sequenceiq/environment/environment/validation/network/azure/AzureEnvironmentNetworkValidator.java

```diff
@@ -65,15 +66,19 @@ public AzureEnvironmentNetworkValidator(CloudNetworkService cloudNetworkService,
     @Override
     public void validateDuringFlow(EnvironmentDto environmentDto, NetworkDto networkDto, ValidationResultBuilder resultBuilder) {
         if (environmentDto == null || networkDto == null) {
-            LOGGER.warn("EnvironmentDto or NetworkDto. Neither them can be null!");
+            LOGGER.warn("EnvironmentDto or NetworkDto. Neither of them could be null!");
             resultBuilder.error("Internal validation error");
             return;
         }
         Map<String, CloudSubnet> cloudNetworks = cloudNetworkService.retrieveSubnetMetadata(environmentDto, networkDto);
         checkSubnetsProvidedWhenExistingNetwork(resultBuilder, networkDto, networkDto.getAzure(), cloudNetworks);
-        checkPrivateEndpointNetworkPoliciesWhenExistingNetwork(networkDto, cloudNetworks, resultBuilder);
-        checkPrivateEndpointsWhenMultipleResourceGroup(resultBuilder, environmentDto, networkDto.getServiceEndpointCreation());
-        checkPrivateEndpointForExistingNetworkLink(resultBuilder, environmentDto, networkDto);
+        if (environmentDto.getStatus() != AVAILABLE) {
```

Comment:
If it is stopped then we need validation right ?

Reply:
They are needed only during env creation, once.

I agree that this is suboptimal, I see 2 options:

1. I can compare against the Env status during the creation (ENVIRONMENT_VALIDATION_IN_PROGRESS)
2. It would be good if we could distinguish clearly if the call was from an "edit" or a "create", I could pass it as a method parameter, but it modifies the interface, too.

Reply:
vote for option 2